crowdsec-docs/docs/appsec/create_rules.md

```diff
@@ -64,7 +64,7 @@ appsec-rules:
 nuclei_template: my-vuln.yaml
 ```
 
-_note:_ as our custom appsec rule is not yet part of the hub, we specify their path relative to the root of the hub directory.
+_note:_ as our custom AppSec rule is not yet part of the hub, we specify their path relative to the root of the hub directory.
```

Comment:
`_note:_ Since our custom AppSec rule has not been added to the hub yet, we need to define its path relative to the root of the hub directory.`